src/vivarium_gates_child_iv_iron/tools/cli.py

```diff
@@ -41,6 +41,10 @@ def make_artifacts(location: str, output_dir: str, append: bool, replace_keys: T
 
 @click.command()
 @click.argument('output_file', type=click.Path(exists=True))
+@click.option('-is', '--include_seeds', 'include_seeds',
```

Comment:
I would change the option to '--include-seeds' (dash, not underscore) to keep it consistent with other options in this codebase

Reply:
Adding to this flag nitpicking to say that having a single dash flag that's more than a character is rare/bad form I think. Do `-s` or get rid of the `-is` flag altogether.